recipe/meta.yaml

```diff
@@ -27,7 +27,18 @@ requirements:
   run:
 
 test:
+  requires:
+    - {{ compiler('c') }}
+    - numpy
+    - python
+  files:
+    - debug_blis.c
   commands:
+    # show CPU features of agent we're running on
+    - python -c "import numpy; numpy._pytesttester._show_numpy_info()"
+    - $CC -I$PREFIX/include -L$PREFIX/lib -lblis -lm -lpthread -lrt debug_blis.c        # [unix]
```

Comment:
```suggestion
    - $CC -I$PREFIX/include -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib -lblis -lm -lpthread debug_blis.c        # [unix]
```

Reply:
sounds like `debug_blis.c` needs to come before that? At least it's not being found anymore:
```
+ /home/conda/feedstock_root/build_artifacts/blis_1625179405412/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/bin/x86_64-conda-linux-gnu-cc -I/home/conda/feedstock_root/build_artifacts/blis_1625179405412/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/include -L/home/conda/feedstock_root/build_artifacts/blis_1625179405412/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib -Wl,-rpath,/home/conda/feedstock_root/build_artifacts/blis_1625179405412/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib -lblis -lm -lpthread debug_blis.c
+ debug_blis
/home/conda/feedstock_root/build_artifacts/blis_1625179405412/test_tmp/run_test.sh: line 9: debug_blis: command not found
```